examples/webgl_mirror.html

```diff
@@ -43,6 +40,9 @@
 
 			let sphereGroup, smallSphere;
 
+			let groundMirrorRenderTarget, verticalMirrorRenderTarget;
```

Comment:
There is no need to cache the render targets in these variables. Just request them with `getRenderTarget()` in `onWindowResize()`.